assets/multiqc_config.yml

```
@@ -1,8 +1,6 @@
 report_comment: >
-
   This report has been generated by the <a href="https://github.com/qbic-pipelines/vcftomaf/releases/tag/1.0.0" target="_blank">qbic-pipelines/vcftomaf</a>
```

```suggestion
  This report has been generated by the <a href="https://github.com/qbic-pipelines/vcftomaf/releases/tag/1.1.0dev" target="_blank">qbic-pipelines/vcftomaf</a>
```

same here for the dev version